stac_fastapi/elasticsearch/tests/extensions/test_filter.py

```diff
@@ -43,6 +46,7 @@ async def test_search_filter_extension_gte(app_client, ctx):
     assert resp.status_code == 200
     assert len(resp.json()["features"]) == 1
 
+    # this part fails
```

Comment:
What's the cause of the failure? Is it a problem? If it's a bad test, it should either be fixed or removed IMO.